package Trycatch;

public class A {
public static void main(String[] args) {
	
	/*System.out.println(1);
	System.out.println(2);
	try {
		System.out.println(4/0);
	}
	
	catch(ArithmeticException P ) {
		System.out.println("YAA");
	}
	System.out.println(5);*/
	
	//HELLO DEAR LET'S GO FOR DINNER
}
}
